akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -348,7 +348,8 @@ private[stream] final class GraphInterpreter(
   val outHandlers: Array[OutHandler], // Lookup table for the outHandler of the connection
   val logics: Array[GraphStageLogic], // Array of stage logics
   val onAsyncInput: (GraphStageLogic, Any, (Any) ⇒ Unit) ⇒ Unit,
-  val fuzzingMode: Boolean) {
+  val fuzzingMode: Boolean,
+  val context: ActorRef) {
```

Comment:
Ah, I see now: it is used as an identifier for the context in which this is running.